test/TritonGPU/accelerate-matmul-cdna1.mlir

```diff
@@ -0,0 +1,800 @@
+// RUN: (! triton-opt %s -split-input-file --tritonamdgpu-accelerate-matmul=arch-generation-name=gfx908 --mlir-pass-pipeline-crash-reproducer=%t 2>/dev/null) | FileCheck --check-prefixes=CHECK %s
```

Comment:
This file is generated,
I will add scripts in next PR, just don't want to add too much things in one PR